src/dmd/lexer.d

```diff
@@ -449,9 +451,11 @@ class Lexer
                 }
                 else if (p[1] == '\"')  // C wide string literal
                 {
+                    const c = *p;
                     ++p;
                     escapeStringConstant(t);
-                    t.postfix = 't';
+                    t.postfix = c == 'L' ? (wchar_tsize == 2 ? 'w' : 'd') :
+                                c == 'u' ? 'w' : 'd';
```

Comment:
please split this nested ternary, it is very hard to follow.

Reply:
ok